### PR TITLE
tiflash: add more explanation for parameter `bg_task_io_rate_limit`

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5739
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@Joyinqin Is this PR ready for review?

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -65,7 +65,12 @@ delta_index_cache_size = 0
 
 ## Storage paths settings take effect starting from v4.0.9
 [storage]
-    ## [Experimental] Introduced in v5.0. Limits the total write rate of background tasks in bytes per second. By default 0, means no limit. It is not recommended to use this experimental feature in a production environment.
+    ## [Experimental] New in v5.0. This item limits the total write rate of background tasks in bytes per second. It is not recommanded to use this experimental feature in a production environment. 
+    ## The unit is byte. Currently, setting such as "10GB" is not supported.
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:69]:
```suggestion
    ## The unit is bytes. Currently, the setting such as "10GB" is not supported.
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -65,7 +65,12 @@ delta_index_cache_size = 0
 
 ## Storage paths settings take effect starting from v4.0.9
 [storage]
-    ## [Experimental] Introduced in v5.0. Limits the total write rate of background tasks in bytes per second. By default 0, means no limit. It is not recommended to use this experimental feature in a production environment.
+    ## [Experimental] New in v5.0. This item limits the total write rate of background tasks in bytes per second. It is not recommanded to use this experimental feature in a production environment. 
+    ## The unit is bytes. Currently, the setting such as "10GB" is not supported.
+    ## The default value is 0, which means no limit.
+    ## This parameter is mainly for the scenario when TiFlash is deployed on AWS EBS (gp2/gp3) disks and is used to control the usage of machine disk bandwidth by background tasks.
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:71]:
```suggestion
    ## This parameter is used to control the usage of machine disk bandwidth by background tasks mainly for the scenario where TiFlash is deployed on the AWS EBS (gp2/gp3) disk.
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -65,7 +65,12 @@ delta_index_cache_size = 0
...
+    ## Improves the stability of TiFlash query performance. The recommended configuration in this scenario is 50% of the disk bandwidth.
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:72]:
```suggestion
    ## This parameter can be used to improve the stability of the TiFlash query performance. The recommended configuration in this scenario is 50% of the disk bandwidth.
```

**Review by lidezhu — APPROVED** [review]
LGTM

**Pull request merged**